resources/language/resource.language.pl_pl/strings.po

```diff
@@ -0,0 +1,761 @@
+# XBMC Media Center language file
+# Addon Name: ExtendedInfo Script
+# Addon id: script.extendedinfo
+# Addon version: 1.0.0
+# Addon Provider: phil65
+msgid ""
+msgstr ""
+"Project-Id-Version: XBMC-Addons\n"
```

Comment:
still looks wrong to me :) I will pull this and remove it myself then. I guess it's something wrong which was copy&pasted over all those years.
Thx.